scipy/spatial/transform/_rotation.py

```diff
@@ -325,13 +328,16 @@ def __init__(
         self._single = quat.ndim == 1 and is_numpy(xp)
         if self._single:
             quat = xpx.atleast_nd(quat, ndim=2, xp=xp)
-        self._backend = backend_registry.get(xp)
+        self._backend = backend_registry.get(xp, xp_backend)
         self._quat: Array = self._backend.from_quat(
             quat, normalize=normalize, copy=copy, scalar_first=scalar_first
         )
 
-    @classmethod
-    def from_quat(cls, quat: ArrayLike, *, scalar_first: bool = False) -> Rotation:
+    @staticmethod
+    @xp_capabilities(
+        skip_backends=[("dask.array", "missing required linalg extension")]
```

Comment:
```suggestion
        skip_backends=[("dask.array", "missing required linalg.cross/det functions")]
```
Slightly misleading: `array_api_compat.dask.array.linalg` exists, but it is incomplete. Please update note below too